src/components/books/InputValue.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../../redux/books/books';
+import './books.css';
+
+const InputValue = () => {
+  const [state, setState] = useState({
+    title: '',
+    author: '',
+    category: 'Fictional',
+  });
+
+  const dispatch = useDispatch();
+
+  const handleChange = (e) => {
+    setState({
+      ...state,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const { title, author, category } = state;
+    if (title.length > 0 && author.length > 0) {
+      dispatch(
+        addBook({
+          title,
+          author,
+          item_id: uuidv4(),
+          category,
+        }),
+      );
+    }
+    state.title = '';
+    state.author = '';
+  };
+
+  return (
+    <div className="input">
+      <h1>ADD NEW BOOK</h1>
+      <form onSubmit={handleSubmit}>
+        <input
+          type="text"
+          name="title"
+          placeholder="Book title..."
+          value={state.title}
+          required
+          onChange={handleChange}
+        />
+        <input
+          type="text"
+          name="author"
+          placeholder="Author..."
+          value={state.author}
+          required
+          onChange={handleChange}
+        />
+        <button className="addBtn" type="submit">
+          ADD BOOK
+        </button>
+      </form>
+    </div>
+  );
+};
```

Comment:
## Your Form
![image](https://user-images.githubusercontent.com/93039359/232180425-e74920ef-3ff3-4d0f-a447-5ec419bf5ecb.png)

## Zeplin's Form
![image](https://user-images.githubusercontent.com/93039359/232181188-325dd443-dadc-4dc5-8f29-37a4be755cdf.png)

- [ ] It would be nice if you make all your form look similar to the Zeplin's design to make your project follow the requirements 😊
1. Kindly add the line which I have highlighted in the above image 👍
2. Kindly make your form expand by adding more widths to input fields and submit button as you can see in the zeplins design 👍